src/components/Sidebar/Sidebar.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.sidebar {  
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: $sidebar-default-width;
+  max-width: $sidebar-max-width;
+  height: $sidebar-default-height;  
+  padding: $sidebar-padding;
+  z-index: $sidebar-z-index;
+  box-shadow: $sidebar-default-box-shadow;
+
+  @include themify($themes) {
+    color: themed('headerColor');
+    background-color: themed('headerBgColor');
+  }
+
+  &__menu-hamburger {
+    font-size: $icon-32;
+    color: $darker-grey;
+    cursor: pointer;
+  }
+  
+  &__menu-hamburger:hover {  
+    color: $black;
+  }
+
+  &__close-btn {
+    display: flex;
+    justify-content: flex-end;
+    font-size: $icon-24;
+    color: $darker-grey;
+    cursor: pointer;
+  }
+
+  &__close-btn:hover {  
+    color: $black;
```

Comment:
same as in comment above. and review all your codes.

Reply:
fixed